.github/workflows/e2e.yml

```diff
@@ -33,7 +33,9 @@ jobs:
       - name: Start server
         run: yarn start:example &
       - name: Run tests
-        run: yarn turbo test:e2e
+        run: |
+          yarn turbo test:e2e 
+          BASE_URL=http://localhost:3000/legacy/admin yarn test:e2e
```

Comment:
E2E tests should be run on both page and app router implementation

Reply:
This is what we are doing here